docs/src/components/home/keyFeaturesSection.tsx

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+
+import { KeyFeatureCard } from "./keyFeatureCard";
+import { PageSection } from "./pageSection";
+
+import EasyToUseImage from "@site/static/assets/home/easy-to-use.png";
+import OpenSourecImage from "@site/static/assets/home/open-source.png";
+import PerformanceImage from "@site/static/assets/home/performance.png";
+
+import "@site/src/css/home/keyFeaturesSection.css";
+
+const easyToUseText =
+	"Transform your collaborative experience with our developer friendly framework - where simplicity meets powerful functionality effortlessly. The framework provides usability that drives innovation within Microsoft and across the industry by dramatically lowering the difficulty and cost of building innovative, collaborative software.";
+
+const openSourceText =
```

Comment:
There are some funky blank characters in several places in this file, just checking if they're intended (maybe non-breaking spaces, or something) or if we should clean up so we don't end up with surprising rendering:

![alt text](image.png)

Reply:
Good catch. Curious how you saw those? Is that a VSCode setting?

Reply:
JK. I see them with my current settings. Not sure how I missed this...